From 35e9355fe21dab3ec2f5865fd23089883518d04b Mon Sep 17 00:00:00 2001
From: Ludovic Barre <ludovic.barre@foss.st.com>
Date: Thu, 30 May 2024 09:51:30 +0200
Subject: [PATCH 0894/1695] product: tfm-stm32mp2: set minimal banner

Set minimal banner to avoid ascii art banner.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 89d00f30601ebceb812ff77dfd03fc83f0ee1410 product: tfm-stm32mp2: set minimal banner]
Change-Id: Idc8b96e37a9617911c97af8ae205351f8b93105b
Signed-off-by: Ludovic Barre <ludovic.barre@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/SCP-firmware/+/383884
Domain-Review: Gatien CHEVALLIER <gatien.chevallier@st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/385197
Tested-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
---
 .../SCP-firmware/product/tfm-stm32mp2/include/fmw_log.h          | 1 +
 1 file changed, 1 insertion(+)

diff --git a/core/lib/scmi-server/SCP-firmware/product/tfm-stm32mp2/include/fmw_log.h b/core/lib/scmi-server/SCP-firmware/product/tfm-stm32mp2/include/fmw_log.h
index 4e0bdd0ee..0898d35f8 100644
--- a/core/lib/scmi-server/SCP-firmware/product/tfm-stm32mp2/include/fmw_log.h
+++ b/core/lib/scmi-server/SCP-firmware/product/tfm-stm32mp2/include/fmw_log.h
@@ -12,5 +12,6 @@
  * Disable log buffering by setting buffer size to 0.
  */
 #define FMW_LOG_BUFFER_SIZE 0
+#define FMW_LOG_MINIMAL_BANNER
 
 #endif /* FMW_LOG_H */
-- 
2.47.2

